web/src/locales/en/data.json

```diff
@@ -175,6 +175,7 @@
     "Please choose the type of your data": "Please choose the type of your data",
     "Please input your search term": "Please input your search term",
     "Prompt": "Prompt",
```

Comment:
It's not manually generated. See: https://casdoor.org/docs/contributing/#how-to-update-i18n-data